modules/ROOT/pages/editor-plugin-version.adoc

```diff
@@ -4,157 +4,156 @@
+==== Using `plugins.min.js`
 
-[#example-using-the-{productmajorversion}-release-channel]
-===== Example: using the `{productmajorversion}` release channel
+In most cases, using `plugins.min.js` is the correct choice. It provides the most flexibility and simplicity by ensuring all plugins are loaded from {companyname} Cloud. If you need to specify individual plugin versions, you can do so via query parameters.
 
-[source,html,subs="attributes+"]
+.Example
+[source,html]
 ----
-<script src="{cdnurl}" referrerpolicy="origin"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/7/plugins.min.js?mentions=3.0&powerpaste=6.0&advcode=sdk" referrerpolicy="origin"></script>
 ----
 
-[#{productmajorversion}-testing-release-channel]
-==== {productmajorversion}-testing release channel
+In this example, {productname} will try to load all premium plugins from {companyname} Cloud, including specified versions for the "mentions" and "powerpaste" plugins. The disadvantage of using `plugins.min.js` is that if you want to load only one plugin from {companyname} Cloud while the rest are self-hosted, you have to add all other plugins as query parameters with their versions specified as "sdk." When {companyname} Cloud releases a new plugin, you'll need to update the list.
 
-This channel deploys the current *release candidate* for the `{productmajorversion}` channel. The {productname} release candidate is undergoing quality assurance. The current version of {productname} available through the `{productmajorversion}-testing` channel can be found on the https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-testing/version.txt[{cloudname} {productname} {productmajorversion}-testing version page].
+include::partial$misc/admon-script-tag-placement.adoc[]
 
-[#example-using-the-{productmajorversion}-testing-release-channel]
-===== Example: using the `{productmajorversion}-testing` release channel
+==== Using `cloud-plugins.min.js`
 
-[source,html,subs="attributes+"]
+{companyname} recommends only using `cloud-plugins.min.js` if you have a specific reason to self-host some premium plugins while loading others from Tiny Cloud. This approach is more complex and requires you to specify which plugins to load from {companyname} Cloud.
+
+.Example
+[source,html]
 ----
-<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/{productmajorversion}-testing/tinymce.min.js" referrerpolicy="origin"></script>
+<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/7/cloud-plugins.min.js?mentions=3.0&powerpaste=6.0&advcode" referrerpolicy="origin"></script>
```

Comment:
Going forward from 7, the versions of plugins should not matter anymore since they are the same as the tinymce version? In this case, what is the version of advcode if the version is not specified?

Reply:
The versions still technically matter, but it's never been clear to me why customers might ever want this. They only do because it's the first example in our docs.

Likewise it's not clear why self-hosted plugins are specified in the `cloud-plugins` url. We probably need to look at what the server actually returns before we proceed much further with this effort.

Reply:
Removed any mention of plugin-versions in last commit, resolving this conversation as its no longer relevant in new approach.